ido/src/modules/testing.rs

```diff
@@ -96,4 +96,28 @@ impl IDOContract{
 
         self.projects.insert(&project_id, &project);
     }
+
```

Comment:
This function should be moved to the unit test emulator. We shouldn't make it as a public smart contract function.